design/one-pager-package-format-v2.md

```diff
@@ -70,17 +77,16 @@ package consumers.
 All software that consumes a package must do so via its output stream - a YAML
 stream emitted to stdout when the package's OCI image entrypoint is invoked. The
 output stream will consist of a `Provider` or `Configuration` YAML document
-(described below), followed by a series of Crossplane-specific YAML documents,
-which must be either:
+(described below) and a series of Crossplane-specific YAML documents, which must
+be either:
 
 * A `CustomResourceDefinition` that defines a Crossplane managed resource.
```

Comment:
Does restricting packages to these types assume that any additional metadata that may be desired in a package, like metadata for a package registry, will be defined in annotations? We've talked about use case involving an icon or readme, are we ok with those being represented as annotation values?

Reply:
> Does restricting packages to these types assume that any additional metadata that may be desired in a package, like metadata for a package registry, will be defined in annotations?

Correct - this document proposes all additional metadata should be annotations on the output stream.

> We've talked about use case involving an icon or readme, are we ok with those being represented as annotation values?

Yes, the idea is those would be annotation values. There were some ideas around using the `--context` flag to the unpack binary to control whether such annotations were actually emitted or not; they're mostly only interesting when indexing a package, and they could push our API objects over the 1MB etcd limit.